src/madpack/madpack.py

```diff
@@ -512,7 +512,9 @@ def _plpy_check(py_min_ver):
         try:
             _internal_run_query("CREATE LANGUAGE plpythonu;", True)
         except:
-            _error('Cannot create language plpythonu. Stopping installation...', False)
+            _error("""Cannot create language plpythonu. Please check if you
+                have configured and installed postgres with `--with-python`
```

Comment:
Can you possibly use "portid" or "args.platform[0]" in place of "postgres"?

Reply:
Sure, that should be more general.